pharmaceuticals_retailers/data/pos_category.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="pos_category_1" model="pos.category">
+        <field name="name">ALLOPATHIC</field>
+    </record>
+    <record id="pos_category_2" model="pos.category">
+        <field name="name">AYURVEDIC</field>
+    </record>
```

Comment:
I'll stop here the review, since this doesn't seem worldwide standard categories. I'll ask a PO to have a look